### PR TITLE
Add vaccine selector for other person

### PR DESCRIPTION
**Pull request by ted107mk**
Adds Their Vaccine selector.

Need clarification on
- Description strings
- Since SegmentedControl's label works differently than SelectControl's, the formatting is inconsistent (whole line is bolded, colon after question mark)

image

### CODE REVIEW

File: src/data/calculate.ts [hunk 1]
```diff
@@ -82,7 +82,7 @@ export const defaultValues: CalculatorData = {
   yourVaccineType: '',
   yourVaccineDoses: 0,
 
-  theirVaccine: 'undefined',
```
Comment (ted107mk) [inline src/data/calculate.ts:85]: I was getting a 'Cannot read property 'value' of undefined' js error from SegmentedControl: https://github.com/microcovid/microcovid/blob/main/src/components/calculator/controls/SegmentedControl.tsx#L57

File: src/data/prepopulated.ts [hunk 2]
```diff
@@ -33,7 +33,7 @@ export const prepopulated: {
     theirMask: '',
     yourMask: '',
     voice: '',
-    theirVaccine: 'undefined',
+    theirVaccine: 'undefined', // TODO update this
```
Comment (ted107mk) [inline src/data/prepopulated.ts:36]: Will update once default value for theirVaccine is resolved

**Review by beshaya — CHANGES_REQUESTED** [review]
Thanks for doing this! Left some cleanup comments.

I just landed the code for importing vaccine prevalence and using it to compute risk. If you rebase onto main, this should work for the "average person" profile.

**Comment by beshaya** [comment]
Also, we'd love to have you on our slack, which would probably make discussing code easier. Could you fill out https://docs.google.com/forms/d/e/1FAIpQLSeF1r6pHvWyIrZ_quJNzChGOOdjdunVAjrcbqj_xbEUbnMTsA/viewform?usp=sf_link if you're interested?

File: src/data/data.ts [hunk 3]
```diff
@@ -388,14 +388,12 @@ export const Vaccines: { [key: string]: VaccineValue } = {
...
-  },
-  undefined: {
-    label: i18n.t('data.their_vaccine.unknown'),
-  },
+const yesVaccinatedDesc = i18n.t('data.yes_vaccinated')
+const noVaccinatedDesc = i18n.t('data.no_vaccinated')
+const unknownVaccinatedDesc = i18n.t('data.unknown_vaccinated')
+
+export const TheirVaccine: { [key: string]: CheckBoxFormValue } = {
+  vaccinated: segmentedFormValue(i18n.t('data.their_vaccine.yes'), yesVaccinatedDesc, 1 / 2),
```
Comment (acconrad) [inline src/data/data.ts:396]: Is this accurate? I thought other discussions had it all the way down to .2-.16

Comment (beshaya) [reply src/data/data.ts:396]: This is a multiplier tedmk found for transmission from vaccinated people who are known to have covid, which is different from the risk reduction from the vaccine.

Comment (acconrad) [reply src/data/data.ts:396]: @beshaya but if, in a given area, a small percentage of people have COVID, wouldn't the absolute risk be far lower than 1/2?

**Comment by beshaya** [comment]
This isn't an absolute risk. This is a multiplier much like wearing a mask.

Also, we aren't going to take this particular multiplier, at least until we
see it corroborated by another study or two and think through whether the
other vaccine multipliers need to be adjusted to avoid double counting
reduced contagiousness.

**Comment by blanchardjeremy** [comment]
Closed in favor of #881

**Pull request closed without merging**